Link error when including OmsAsyn support in a Windows IOC

The following link errors occur when when including OmsAsyn support in a Windows IOC (EPICS_HOST_ARCH=windows-x64-static, VS 2015):
```
iockmp2Win64_registerRecordDeviceDriver.obj : error LNK2019: unresolved external symbol pvar_func_OmsMAXnetAsynRegister referenced in function iockmp2Win64_registerRecordDeviceDriver
iockmp2Win64_registerRecordDeviceDriver.obj : error LNK2019: unresolved external symbol pvar_func_OmsMAXvAsynRegister referenced in function iockmp2Win64_registerRecordDeviceDriver
```

--- a/motorApp/OmsAsynSrc/omsMAXnet.cpp
+++ b/motorApp/OmsAsynSrc/omsMAXnet.cpp
@@ -363,6 +363,9 @@ extern "C" int omsMAXnetConfig(
 
 /* Code for iocsh registration */
 
+extern "C"
+{
+
 /* omsMAXnetConfig */
 static const iocshArg omsMAXnetConfigArg0 = {"asyn motor port name", iocshArgString};
 static const iocshArg omsMAXnetConfigArg1 = {"number of axes", iocshArgInt};
@@ -389,3 +392,4 @@ static void OmsMAXnetAsynRegister(void)
 
 epicsExportRegistrar(OmsMAXnetAsynRegister);
 
+}

--- a/motorApp/OmsAsynSrc/omsMAXv.cpp
+++ b/motorApp/OmsAsynSrc/omsMAXv.cpp
@@ -589,6 +589,10 @@ extern "C" int omsMAXvConfig2(
 }
 
 /* Code for iocsh registration */
+
+extern "C"
+{
+
 /* omsMAXvSetup */
 static const iocshArg setupArg0 = {"Max. controller count", iocshArgInt};
 static const iocshArg setupArg1 = {"VME address type", iocshArgInt};
@@ -649,3 +653,4 @@ static void OmsMAXvAsynRegister(void)
 
 epicsExportRegistrar(OmsMAXvAsynRegister);
 
+}

